### ISSUE TITLE
Wrong position for edges if direction: RTL

### ISSUE THREAD

**Issue by ahmadi3d** [wbkd/react-flow#3116]
### Describe the Bug

I tried the following code which is the first sample code in the Quick Start page but the edges are in the wrong position if the direction is RTL, If I change the direction to LTR this is fixed, does the component support RTL ? How can I fix this issue, Thank you!
```
import React from 'react';
import ReactFlow from 'reactflow';

import 'reactflow/dist/style.css';

const initialNodes = [
  { id: '1', position: { x: 0, y: 0 }, data: { label: '1' } },
  { id: '2', position: { x: 0, y: 100 }, data: { label: '2' } },
];
const initialEdges = [{ id: 'e1-2', source: '1', target: '2' }];

export default function App() {
  return (
    <div style={{ width: '100vw', height: '100vh' }}>
      <ReactFlow nodes={initialNodes} edges={initialEdges} />
    </div>
  );
}
```

### Steps to Reproduce the Bug or Issue

1- Give your react app or HTML body direction: RTL
2- Try the first example in the Quick start
3- You can see the edges are in wrong location

### Expected behavior

It seems the nodes are in the right place so the edges should be too.

### Screenshots or Videos

![image](https://github.com/wbkd/react-flow/assets/20884578/9d1eb4df-a3eb-4669-a181-55c394b61a9d)

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by npirotte** [comment]
I'm taking the opportunity to ask about something that might be related.

I'm also building some flow diagrams, embedded within a remix application.

The flow of my diagrams can be both LTR or RTL depending of some context.

When the flow is RTL, I'm encountering a slight miss placement of the edge end (see screenshot)
This is RTL:
<img width="288" alt="Screenshot 2023-06-12 at 11 13 38" src="https://github.com/wbkd/react-flow/assets/2330137/6f5ad8ca-e1c6-409e-b955-1b1b783cb76a">

This is LTR
<img width="178" alt="Screenshot 2023-06-12 at 11 14 00" src="https://github.com/wbkd/react-flow/assets/2330137/b9c16825-e418-4d43-b350-07d11158a2dd">

The placement of the handles is correct, but you can see that the placement of the arrow if off of a few px.

The strangest thing is that this doesn't happen consistently.
* When I directly load the page, it's ok
* If I navigate to the page from another page, then it's not ok.

Any help would be welcome :)

**Comment by npirotte** [comment]
Well after more testing, in my case, there seems to be a interference between the dialog opening transition and the layout computation. Delaying the rendering of the graph helped.

**Comment by moklick** [comment]
@ahmadi3d setting the direction of the flow itself (".react-flow") to ltr should solve your issue.

**Comment by devnull69** [comment]
I have the same problem. I've set the body dir to "rtl" and expected the graph to still show correctly. I would take care of all orientation related issues myself, but the edge connection is off.

Setting `dir="rtl"` on the `.react-flow` element didn't fix it

**Comment by devnull69** [comment]
To be honest ... I think the edges are "correct", so they are placed correctly. But rather the node positions are off. They don't match the expected coordinates.

**Comment by moklick** [comment]
This is fixed in 11.11.1

**Comment by devnull69** [comment]
I verified it works with 11.11.1

Thanks!
